3.process-cfret-features/3.process-cfret-features.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
-# convert jupyter notebook to a .py file once all variable changes are made
+# convert single cell jupyter notebook to a .py file once all variable changes are made
 jupyter nbconvert --to python extract_sc_features.ipynb
```

Comment:
Consider using the below statement to convert notebook (into `nbconverted/scripts` folder) and execute the converted file in one line:

```
jupyter nbconvert --to python \
        --FilesWriter.build_directory=scripts/nbconverted \
        --execute extract_sc_features.ipynb
```

Same comment for the next `nbconvert`/`python` lines below

Reply:
This is much better! I have implemented this.